source/dotnet/Library/AdaptiveCards.Net6/AdaptiveCards.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
I'm curious -- did you try adding `net6.0` to the existing .csproj's `TargetFrameworks`? It would be nice if we could ship this as an additive thing rather than a separate package.

Reply:
we're releasing in the same package, we're just adding both dlls to the same nuget package

Reply:
if it's _possible_ to get `net6.0` building in the existing project, it would be nice to not have to duplicate all of this scaffolding. I'll resolve this and sign off, but please do give it a shot 👍🏼